src/main/java/net/seakerman/inventoryfullalert/config/ModConfig.java

```diff
@@ -0,0 +1,36 @@
+package net.seakerman.inventoryfullalert.config;
+
+import me.shedaniel.autoconfig.ConfigData;             // autoconfig is better for settings menu
+import me.shedaniel.autoconfig.annotation.Config;      // easy-to-read code and less files
+import me.shedaniel.autoconfig.annotation.ConfigEntry;
+
+@Config(name = "inventoryfullalert")
```

Comment:
Completely new config script using Autoconfig which is more easier to code with same possibilities.